miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/component/InitBinderAdvice.java

```diff
@@ -0,0 +1,36 @@
+package uk.ac.bbsrc.tgac.miso.webapp.controller.component;
+
+import static uk.ac.bbsrc.tgac.miso.core.util.LimsUtils.*;
+
+import java.beans.PropertyEditorSupport;
+import java.math.BigDecimal;
+
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.InitBinder;
+
+@ControllerAdvice("uk.ac.bbsrc.tgac.miso.webapp.controller")
+public class InitBinderAdvice {
+
+  @InitBinder
+  public void initBinder(WebDataBinder binder) {
+    binder.registerCustomEditor(Long.class, new PropertyEditorSupport() {
+      @Override
+      public void setAsText(String text) throws IllegalArgumentException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)